rust/Cargo.toml

```diff
@@ -9,8 +9,8 @@ resolver = "2"
 
 [workspace.dependencies]
 log = "0.4.8"
-zingolib = { git="https://github.com/zingolabs/zingolib", default-features=true, branch = "dev" }
-zingoconfig = { git="https://github.com/zingolabs/zingolib", default-features=true, branch = "dev" }
+zingolib = { git="https://github.com/Oscar-Pepper/zingolib", default-features=true, branch = "darkside_helpers" }
```

Comment:
I think you need to change this back to `zingolabs` & `dev`.

Reply:
yeah i havent cleaned up for merge yet, now darkside_helpers is merged i can put this back to zingolabs dev. thanks